src/core/QCheck2.mli

```diff
@@ -1660,6 +1669,18 @@ module Test : sig
       See {!make_cell} for a description of the parameters.
   *)
 
+  val make_neg :
+    ?if_assumptions_fail:([`Fatal | `Warning] * float) ->
+    ?count:int -> ?long_factor:int -> ?max_gen:int -> ?max_fail:int -> ?retries:int ->
+    ?name:string -> ?print:('a Print.t) -> ?collect:('a -> string) -> ?stats:('a stat list) ->
+    'a Gen.t -> ('a -> bool) -> t
+  (** [make_neg gen prop] builds a test that checks property [prop] on instances
+      of the generator [gen].
+      The test is considered negative, meaning that it is expected to fail.
```

Comment:
"expected to fail" makes me think of actual failures. Would something like "does not meet the property" be better? I'm not sure, you can ignore this comment

Reply:
I think this is a relevant concern to help distinguish a "failed property" from a "failed test".
In 9199574 I've changed the formulation to "expected not to satisfy the tested property" in both the QCheck and QCheck2 interfaces to help clarify.

Reply:
It's better yes, thank you. Could you also modify the changelog with your modification?

Reply:
Fixed in daa57f8